app/packages/core/src/components/Actions/similar/Similar.tsx

```diff
@@ -33,24 +34,25 @@ interface SortBySimilarityProps {
   bounds?: any; // fix me
 }
 
-export const searchBrainKeyValue = atom<string>({
-  key: "searchBrainKeyValue",
-  default: "",
-});
-
 const SortBySimilarity = ({
   modal,
   bounds,
   close,
   isImageSearch,
 }: SortBySimilarityProps) => {
   const current = useRecoilValue(fos.similarityParameters);
+  const datasetId = useRecoilValue(fos.dataset).id;
+  const [lastUsedBrainkeys, setLastUsedBrainKeys] =
```

Comment:
nit: camelcase error in `lastUsedBrainkeys`, capitalize `k`. Consider removing the setter since it's not used